pinot-core/src/test/java/com/linkedin/pinot/core/data/partition/PartitionFunctionTest.java

```diff
@@ -43,10 +43,42 @@ public void testModulo() {
       String partitionFunctionString = "MoDuLo" + PartitionFunctionFactory.PARTITION_FUNCTION_DELIMITER + divisor;
       PartitionFunction partitionFunction = PartitionFunctionFactory.getPartitionFunction(partitionFunctionString);
 
+      Assert.assertEquals(partitionFunction.toString(),
+          "modulo" + PartitionFunctionFactory.PARTITION_FUNCTION_DELIMITER + divisor);
+
       for (int j = 0; j < 1000; j++) {
         int value = random.nextInt();
         Assert.assertEquals(partitionFunction.getPartition(value), (value % divisor));
       }
     }
   }
+
+  /**
+   * Unit test for {@link DefaultKafkaPartitionFunction}.
+   * <ul>
+   *   <li> Tests that partition values are in expected range. </li>
+   *   <li> Tests that toString returns expected string. </li>
+   * </ul>
+   */
+  @Test
+  public void testDefaultKafkaPartitioner() {
+    long seed = System.currentTimeMillis();
+    Random random = new Random(seed);
+
+    for (int i = 0; i < 1000; i++) {
+      int divisor = Math.abs(random.nextInt());
```

Comment:
Returns a pseudorandom, uniformly distributed int value between 0 (inclusive) and the specified value (exclusive), drawn from this random number generator's sequence.
Need to handle divide by 0 or exclude 0